src/components/enterprise-user-subsidy/enterprise-offers/data/hooks.js

```diff
@@ -50,10 +51,11 @@ export const useEnterpriseOffers = ({
       setCanEnrollWithEnterpriseOffers(true);
     }
 
-    const currentEnterpriseOffers = enterpriseOffers.filter(offer => offer.isCurrent);
-    if (currentEnterpriseOffers.length > 0) {
-      const hasLowBalance = currentEnterpriseOffers.some(offer => offer.isLowOnBalance);
-      const hasNoBalance = currentEnterpriseOffers.every(offer => offer.isOutOfBalance);
+    const currentEnterpriseOffersList = enterpriseOffers.filter(offer => offer.isCurrent);
+    if (currentEnterpriseOffersList.length > 0) {
+      setCurrentEnterpriseOffers(currentEnterpriseOffersList);
```

Comment:
[inform] Similar logic as introduced in https://github.com/openedx/frontend-app-learner-portal-enterprise/pull/894